include/sys/zio_impl.h

```diff
@@ -48,6 +49,7 @@ extern "C" {
  * Free:
  * Claim:
  * Ioctl:
+ * Trim:
```

Comment:
nit: should be added to line 43 as well.

Reply:
Hah, nice. Done.